src/test/java/testsuite/integration/utility/AuroraTestUtility.java

```diff
@@ -0,0 +1,293 @@
+     */
+    public String initCluster(String username, String password, String identifier)
+        throws InterruptedException {
+        dbUsername = username;
+        dbPassword = password;
+        dbIdentifier = identifier;
+        return initCluster();
+    }
+
+    /**
+     * Creates RDS Cluster/Instances and waits until they are up, and proper IP whitelisting for databases.
+     * @return An endpoint for one of the instances
+     * @throws InterruptedException when clusters have not started after 30 minutes
+     */
+    public String initCluster() throws InterruptedException {
+        // Create Cluster
+        CreateDBClusterRequest dbClusterRequest = new CreateDBClusterRequest()
+            .withDBClusterIdentifier(dbIdentifier)
+            .withMasterUsername(dbUsername)
+            .withMasterUserPassword(dbPassword)
+            .withSourceRegion(dbRegion)
+            .withEnableIAMDatabaseAuthentication(true)
+            .withEngine(dbEngine)
+            .withStorageEncrypted(true);
+
+        rdsClient.createDBCluster(dbClusterRequest);
+
+        // Create Instances
+        CreateDBInstanceRequest dbInstanceRequest = new CreateDBInstanceRequest()
+            .withDBClusterIdentifier(dbIdentifier)
+            .withDBInstanceClass(dbInstanceClass)
+            .withEngine(dbEngine)
+            .withPubliclyAccessible(true);
+
+        for (int i = 1; i <= numOfInstances; i++) {
+            rdsClient.createDBInstance(dbInstanceRequest.withDBInstanceIdentifier(dbIdentifier + "-" + i));
+        }
+
+        // Wait for all instances to be up
+        AmazonRDSWaiters waiter = new AmazonRDSWaiters(rdsClient);
+        DescribeDBInstancesRequest dbInstancesRequest = new DescribeDBInstancesRequest()
+            .withFilters(new Filter().withName("db-cluster-id").withValues(dbIdentifier));
+        Waiter<DescribeDBInstancesRequest> instancesRequestWaiter = waiter
+            .dBInstanceAvailable();
+        Future<Void> future = instancesRequestWaiter.runAsync(new WaiterParameters<>(dbInstancesRequest), new NoOpWaiterHandler());
+        try {
+            future.get(30, TimeUnit.MINUTES);
+            addRunnerIP();
+        } catch (WaiterUnrecoverableException | WaiterTimedOutException | TimeoutException | ExecutionException exception) {
+            teardown();
+            throw new InterruptedException("Unable to start AWS RDS Cluster & Instances after waiting for 30 minutes");
+        } catch (UnknownHostException exception) {
+            throw new InterruptedException("Unable to authorize runner IP");
+        }
+
+        DescribeDBInstancesResult dbInstancesResult = rdsClient.describeDBInstances(dbInstancesRequest);
+        return dbInstancesResult.getDBInstances().get(0).getEndpoint().getAddress();
+    }
+
+    /**
+     * Adds current runner's public IP to EC2 Security groups for RDS access.
+     * @throws UnknownHostException
+     */
+    private void addRunnerIP() throws UnknownHostException {
```

Comment:
If I need to use this utility on my local workstation, this method name will confuse me. Maybe it makes sense to split parts about getting current IP address and whitelisting it.